helpers/error_helper.go

```diff
@@ -0,0 +1,159 @@
+package helpers
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/leechongyan/Studtor_backend/constants"
+)
+
+type RequestError struct {
+	StatusCode int
+
+	Err error
+}
+
+func (r *RequestError) Error() string {
+	return fmt.Sprintf("status %d: %v", r.StatusCode, r.Err)
```

Comment:
i thikn this should be `r.Err.Error()`?

Reply:
We can do so but i am printing by its value so is not necessary to give it string form